Esri/arcgis-maps-sdk-swift-samples
# [New] Show popup

## Description

This PR implements `Show popup` in `Display information` category.
URL to README: [URL](https://github.com/Esri/arcgis-maps-sdk-swift-samples/blob/des12437/Show-popup/Shared/Samples/Show%20popup/README.md)

## Linked Issue(s)

- `swift/issues/3579`

## How To Test
- Tap any feature on the map to see a popup with the attributes of the feature listed.
- Tap the close icon or swipe down on the popup view to dismiss the popup.

## Screenshots
<img width="350" alt="" src="https://user-images.githubusercontent.com/117859673/227063181-6e78dfd7-97d9-452b-9f05-1a4b5fc5d4b0.png">
